_content/articles/utopian-thought-in-tibetan-buddhism_kollmar-paulenz.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Utopian Thought in Tibetan Buddhism: A Survey of the Śambhala Concept and its Sources"
+authors:
+  - "Karénina Kollmar-Paulenz"
+external_url: "https://buddhism.lib.ntu.edu.tw/en/search/search_detail.jsp?seq=295058&comefrom=authorinfo"
+drive_links:
+  - "https://drive.google.com/file/d/1Lum1F8clFY_OAP3TE6_NzG9WesAlI7DQ/view?usp=sharing"
+course: tibetan
+tags:
+  - mythology
```

Comment:
Let's go with `ideology` instead, as it has some ramifications for the interplay between religion and political mythology